src/app/app-routing.module.ts

```diff
@@ -2,11 +2,13 @@ import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 import { HomeComponent } from './home/home.component';
 import { PageNotFoundComponent } from './page-not-found/page-not-found.component';
+import { SigningComponent } from './signing/signing.component';
 
 const routes: Routes = [
   { path: '', component: HomeComponent },
-  { path: 'page-not-found', component: PageNotFoundComponent },
-  { path: '**', pathMatch: 'full', redirectTo: '/page-not-found'}
+  { path: 'pagenotfound', component: PageNotFoundComponent },
+  { path: 'signing', component: SigningComponent },
+  { path: '**', pathMatch: 'full', redirectTo: '/pagenotfound'}
```

Comment:
You're using the incorrect routerLink for PNF. 
```suggestion
  { path: '**', pathMatch: 'full', redirectTo: '/page-not-found'}
```